packages/Bio/src/tests/mm-distance-tests.ts

```diff
@@ -120,7 +120,7 @@ category('Distance', async () => {
       const seq1 = Array(10000).fill('FWRY').join('');
       const seq2 = Array(10000).fill('FYWRRY').join('');
       _testDistance(seq1, seq2, df, -20000);
-    } else _testDistance(prot5, prot6, df, 1);
+    } else { _testDistance(prot5, prot6, df, 1); }
```

Comment:
Put expressions in if-else on the next line and don't use brackets if it's a single line.

Reply:
generally yes. but this is according to what is found in packages eslint config, which dictates that braces should be consistent. i.e. if the if statement on line 119 has braces, so should the consequent elses. this rule was there before me.